File: MyTournamentAPI/OurTournamentAPI/Models/TorneoParticipacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OurTournamentAPI.Models
{
    public class TorneoParticipacion:Torneo
    {
        private int IDParticipacion;
        private int iDEquipo;

        public int IDParticipacion1 { get => IDParticipacion; set => IDParticipacion = value; }
        public int IDEquipo { get => iDEquipo; set => iDEquipo = value; }

        public TorneoParticipacion(int idtorneo, string nombreTorneo, string contraseniaDeAdministrador, string linkParaUnirse, int idparticipacion, int idequipo) :
            base( idtorneo,  nombreTorneo,  contraseniaDeAdministrador,  linkParaUnirse)
        {
            this.IDParticipacion = idparticipacion;
            this.IDEquipo = idequipo;
        }
        public TorneoParticipacion() : this(0, "", "", "", 0,0)
        {
        }

      
    }
}

File: MyTournamentApp/app/src/main/java/com/example/ourtournament/Objetos/TorneoParticipacion.java
package com.example.ourtournament.Objetos;

public class TorneoParticipacion extends Torneo{
    public int IDParticipacion1;
    public int IDEquipo;
    public TorneoParticipacion(int idtorneo, String nombreTorneo, String contraseniaDeAdministrador, String linkParaUnirse, int idparticipacion,int idequipo)
    {
        super(idtorneo, nombreTorneo,  contraseniaDeAdministrador, linkParaUnirse);
        IDParticipacion1 = idparticipacion;
        IDEquipo = idequipo;
    }

    public TorneoParticipacion()
    {
        this(0,"","","",0,0);
    }
}
